test/NeoSharp.Persistence.Redis.Tests/UtRedisDbBinaryRepository.cs

```diff
@@ -14,14 +14,14 @@
 namespace NeoSharp.Persistence.Redis.Tests
 {
     [TestClass]
-    public class UtRedisDbRepository : TestBase
+    public class UtRedisDbBinaryRepository : TestBase
     {
         [TestMethod]
         public async Task GetIndexHeight_NoValueFound_ReturnsZero()
```

Comment:
Maybe we can join all the unit test of IRepository together, like in a foreach?

Reply:
You mean to merge all Ut___Repository classes and merge the methods?
Rocks mocking is really different than Redis mocking, so it will probably makes the code less organised and readable. Don't you think?